### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: distribution/testing.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ Clear the cache of the `test` environment:
 
 Then run:
 
-    $ docker-compose run --rm app vendor/bin/behat`.
+    $ docker-compose run --rm app vendor/bin/behat
```
Comment (soyuka) [inline distribution/testing.md]: Missing a code block quote at the beginning no?

Comment (mickaelandrieu) [inline distribution/testing.md]: The rendering looks good => View button

**Comment by mickaelandrieu** [comment]
@dunglas are we ok on this pull request? Does it need to be improved?

**Comment by Simperfit** [comment]
I think it needs to be done vs 2.0.

File: distribution/testing.md (hunk 1 above)
Comment (teohhanhui) [inline distribution/testing.md]: This is fine.

**Comment by dunglas** [comment]
Thanks @mickaelandrieu

**Pull request merged**